test/test_files/sbol3_genbank_conversion/test_extra_properties_with_references.gb

```diff
@@ -18,53 +18,26 @@ REFERENCE   1  (bases 1 to 5485)
 REFERENCE   2  (bases 1 to 5485)
   AUTHORS   Choi,Y.H., Choi,S.B. and Cho,S.H.
   TITLE     Direct Submission
-  JOURNAL   Submitted (06-JUL-1999) Biology, Inha University, Yonghyon-Dong
-            253, Inchon 402-751, Korea
+  JOURNAL   Submitted (06-JUL-1999) Biology, Inha University, Yonghyon-Dong 253,
+            Inchon 402-751, Korea
 FEATURES             Location/Qualifiers
-     source          1..5485
-                     /label="c1"
```

Comment:
This looks like you're losing a bunch of feature information: is this intentional for testing purposes, or accidental?

Reply:
Yes, since for this particular unit test, I intended to test just the storage of extra properties and reference info, I removed the Feature Qualifiers, as the converter doesn't yet store them while parsing, and the tests would fail on the round trip due to this.
This should be fine, right? I can add other test files when the converter is able to store Feature Qualifiers as well.

Reply:
Yes, that's fine --- as noted above, just please make sure the description text in the header makes it clear this is a modified version of the file.

Reply:
Okay, just to confirm, by description, you mean the Definition section (2nd line), right?

Reply:
Yes, sorry: Definition is what I meant.